docs/api.rst

```diff
@@ -16,6 +16,13 @@ Adaptador Base
 Adaptadores
 -----------
 
+CpiAllUrbanCityAverage (FED's Consumer Price Index for All Urban Consumers: All Items (CpiAllUrbanCityAverage))
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
The underline doesn't have the same number of the characters of the text it's underlining. Can we fix that?